internal/bgp/frr/frr_test.go

```diff
@@ -241,33 +239,6 @@ func TestSingleAdvertisementNoRouterID(t *testing.T) {
 	testCheckConfigFile(t)
 }
 
-func TestSingleAdvertisementInvalidPrefix(t *testing.T) {
```

Comment:
Why delete this one?

Reply:
Because it has no value anymore, since the only validation is done against the length of the communities.

Reply:
:+1: Do you need to delete the related golden file?

Reply:
done